cms/djangoapps/contentstore/tests/test_i18n.py

```diff
@@ -70,11 +70,7 @@ def setUp(self):
         self.course = CourseFactory.create()
         self.field_data = mock.Mock()
```

Comment:
Nit: does `self.field_data` still get used in some way?

Reply:
Only in the `XMLModuleStore`, where it's declared explicitly:
https://github.com/openedx/edx-platform/blob/f13263df4f0e9aff48fbbb7ca04583619317b947/xmodule/modulestore/xml.py#L368-L369

I removed this in 1d42e77.